src/rdkafka_admin.c

```diff
@@ -4346,6 +4411,16 @@ rd_kafka_DescribeAclsResponse_parse(rd_kafka_op_t *rko_req,
                         rd_kafka_buf_read_i8(reply, &resource_pattern_type);
                 }
 
+                if (res_type <= RD_KAFKA_RESOURCE_UNKNOWN ||
```

Comment:
Maybe we should emit a warning log here if the broker returns an (for us) unknown resource or pattern type, something like:
`rd_rkb_log(rkb, LOG_WARNING, "ACLRESPONSE", "DescribeAclsResponse returned unknown resource type %d");`